pilot/pkg/features/pilot.go

```diff
@@ -70,7 +70,7 @@ var (
 
 	DebounceMax = env.RegisterDurationVar(
 		"PILOT_DEBOUNCE_MAX",
-		10*time.Second,
+		5*time.Second,
```

Comment:
if you think the debounce default max requires a change, could you break out this one liner as a separate PR?

Reply:
No. I do not intend to change this. I am experimenting with few values for some of the sensitive tests